sycl/gdb/libsycl.so-gdb.py

```diff
@@ -324,6 +325,12 @@ def to_string(self):
             # error message otherwise. Individual array element access failures
             # will be caught by iterator itself.
             _ = self.value["common_array"]
+            if self.value.type.code == gdb.TYPE_CODE_REF:
+                return "(%s &) @%s: %s" % (
```

Comment:
Can we use `"{} {}".format(a, b)` syntax instead?

Reply:
Yes, I changed it now.